### PR TITLE
[BIOMAGE-1930] Remove account specific hardcodings in nodejs chart

### PR DESCRIPTION
**Pull request by StefanBabukov**
# Background
#### Link to issue 
https://biomage.atlassian.net/browse/BIOMAGE-1930
#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock  (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/hms-dbmi-cellenics/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/hms-dbmi-cellenics/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/hms-dbmi-cellenics/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: .github/workflows/deploy-infra.yaml [hunk 1]
```diff
@@ -238,6 +238,7 @@ jobs:
           yq w -d0 infra/account-config.yaml myAccount.domainName --style=double "$DOMAIN_NAME" \
           | yq w -d0 - myAccount.region --style=double "$AWS_REGION" \
           | yq w -d0 - myAccount.accountId --style=double "$AWS_ACCOUNT_ID" \
+          | yq w -d0 - myAccount.acmCertificate --style=double "arn:aws:acm:eu-west-1:242905224710:certificate/de2efba9-8964-49d2-8525-242e7e43ca4c" \
```
Comment (aerlaut) [inline .github/workflows/deploy-infra.yaml:241]: Should we store this in Github secrets?

File: charts/nodejs/templates/kubernetes-event-exporter.yaml [hunk 2]
```diff
@@ -42,10 +42,8 @@ data:
     receivers:
       - name: "cleanup"
         webhook:
-{{ if eq .Values.kubernetes.env "production" }}
-          endpoint: https://api.scp.biomage.net/v2/kubernetesEvents
-{{ else }}
-          endpoint: https://api-{{ .Values.biomageCi.sandboxId }}.scp-staging.biomage.net/v2/kubernetesEvents
+          endpoint: https://{{ template "hostname" .}}/v2/kubernetesEvents
+
 {{ end }}
```
Comment (aerlaut) [inline charts/nodejs/templates/kubernetes-event-exporter.yaml]: Remove this

File: .github/workflows/deploy-infra.yaml (hunk 1 above)
Comment (aerlaut) [reply .github/workflows/deploy-infra.yaml:241]: What do you think? @ivababukova

Comment (ivababukova) [reply .github/workflows/deploy-infra.yaml:241]: we will, it's ok in here for now

Comment (ivababukova) [reply .github/workflows/deploy-infra.yaml:241]: we can store it in Github secrets either as part of this task or as part of one of the next ones, happy either way.

**Review by kafkasl — APPROVED** [review]

**Pull request merged**